doc/guides/west/repo-tool.rst

```diff
@@ -205,11 +205,11 @@ important to understand.
      revision using ``--mr``. For example, invoking west with: ``west init -m
      https://github.com/zephyrproject-rtos/zephyr --mr v1.15.0`` would clone
      the upstream official zephyr repository at the tagged release v1.15.0
-     (``-m`` defaults to https://github.com/zephyrproject-rtos/zephyr, and
-     ``--mr`` defaults to ``master``).
+     (``-m`` defaults to https://github.com/zephyrproject-rtos/zephyr``, and
```

Comment:
Missing double backtick before `https://`

Reply:
I'm going to treat this as "extra backticks after zephyr", i.e. remove the trailing backticks so the link renders.